index.js

```diff
@@ -3,30 +3,84 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+let flag = true
+if(rolls.length === 0) {
+  return true
+}
+  for (let i = 0; i < rolls.length; i++) {
+    if(typeof rolls[i] === 'number') {
```

Comment:
returning true in this conditional means that if the first element in the array is a number it will automatically return true and stop executing the rest of loop or code in your function.